node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala

```diff
@@ -531,6 +531,13 @@ final case class Options(arguments: Seq[String]) extends ScallopConf(arguments)
       descr = "Private key for dummy deploys."
     )
 
+    // Similar value `casper.min-phlo-price` defined in defaults.conf and should be moved to Scala based config
+    val minPhloPrice = opt[Long](
+      descr = "MinPhloPrice",
+      default = Some(1),
```

Comment:
Do we need default here, it should be populated from Hocon file default.conf when merged with CLI options?

Reply:
Yes, you are right. Default value merges to NodeConf from defaults.conf. Fixed in [commit](https://github.com/rchain/rchain/pull/3556/commits/1864881c2e01aed1c025bf31dcddfa01b9bf30a5).